modules/arm_plugin/src/opset/conv_arm.hpp

```diff
@@ -21,7 +21,8 @@ class ArmConvolution : public Convolution {
                    const ngraph::CoordinateDiff& pads_begin,
                    const ngraph::CoordinateDiff& pads_end,
                    const ngraph::Strides& dilations,
-                   const ngraph::op::PadType& auto_pad);
+                   const ngraph::op::PadType& auto_pad,
+                   const ngraph::PartialShape& output_shape = {});
```

Comment:
in what case can this output_shape be empty?

Reply:
like here where Convolution is converted to ArmConvolution:  https://github.com/openvinotoolkit/openvino_contrib/blob/c9a60a4d4aa7939595fc451010e5b9b8a545af8b/modules/arm_plugin/src/transformations/conv_bias_fusion.cpp#L158-L163

Reply:
I believe we can replace this transformation with `ConvertArmConvolutionLayout`